### ISSUE TITLE
Strange behaviour of ssowatconf

### ISSUE THREAD

**Issue by mbugeia** [YunoHost/moulinette-yunohost#52]
In a yunohost app package I can do soemething like:
`yunohost app setting myapp unprotected_uris -v "/"`
`yunohost app ssowatconf`
That will result in a unprotected_urls ssowat parameter (urls and not uris). 

But if I do that:
`yunohost app setting myapp unprotected_urls -v "/"`
`yunohost app ssowatconf`
That will result in no configuration of ssowat at all! That's confusing because it seems that the correct ssowat configuration is with urls (in https://github.com/Kloadut/SSOwat readme).

Same with:
skipped_urls
protected_urls

Here is the faulty code IMHO:
https://github.com/YunoHost/moulinette-yunohost/blob/e93f076856a8292003865be35d4b6a043121be1d/lib/yunohost/app.py#L899-L926

**Comment by Kloadut** [comment]
Yep that's actually the desired behavior, but with a really poor wording.

We differentiate "URI" and "URL". The SSOwat configuration has `*_urls` parameters, and the yunohost's apps have `*_uris`.
This is because with thos URIs, yunohost will compute full URLs to protect.


For example, if I install an app (`myapp`) on "mydomain.org/app" and I want it unprotected, I can do this by running:
```
 yunohost app setting myapp unprotected_uris -v "/"
```

This is unrelated to the full URL though. And YunoHost will know how to build and save the right path in the SSOwat configuration.

**Comment by mbugeia** [comment]
Ok, thanks I get it now! That was confusing because there is no mention of it in yunohost doc, I will add it. Page for reference: https://yunohost.org/#/packaging_apps

Maybe it would be nice to have the ability to add actual unprotected_urls too. I ran into this problem when trying to package seafile, for example I must unprotect domain.tld/seafhttp but the app is installed on another path. Currently the only way to do that is to edit /etc/ssowat/conf.json.persistent but we lose the flexibility of apps.
